packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -354,14 +354,19 @@ export function makeCollectionManager(
       }
     }
 
+    function mustGet(key, label) {
+      if (passStyleOf(key) === 'remotable' && getOrdinal(key) === undefined) {
+        failNotFound(key, label);
+      }
+      const dbKey = keyToDBKey(key);
+      const result = syscall.vatstoreGet(dbKey) || failNotFound(key, label);
+      return { dbKey, result };
+    }
+
```

Comment:
this will be a good starting point for a future refactoring that avoids the duplicate vatstoreGet syscall